db/schema.sql

```diff
@@ -0,0 +1,23 @@
+DROP DATABASE IF EXISTS banner_images;
+
+CREATE DATABASE banner_images;
+
+USE banner_images;
+
+CREATE TABLE listing (
+  id int(100) NOT NULL AUTO_INCREMENT,
+  PRIMARY KEY (ID)
+);
+
+CREATE TABLE images (
+  id int NOT NULL AUTO_INCREMENT,
+  url varchar(2083) NOT NULL,
+  description varchar(255),
+  user_submit boolean,
+  date date,
```

Comment:
Have you tried to see if this works yet? I think date is a reserved variable so you might need a different name

Reply:
fixed it!